scripts/us_fed/treasury_constant_maturity_rates/extract_data.py

```diff
@@ -0,0 +1,41 @@
+USAGE = '''
+python3 extract_data.py [path]
+    "path": Path to the raw csv containing rates at all maturities.
+
+Example: python3 extract_data.py FRB_H15.csv
+    This will extract the data out of the raw csv and store it in
+    "treasury_constant_maturity_rates.csv".
+    The output table has the same number of columns as the number of constant
+    maturities provided and an extra column for dates.
+    "date" column is of the form "YYYY-MM-DD".
+    The other interest rate columns are numeric.
+'''
+
+
+import sys
+
+from maturity import MATURITIES
+import pandas as pd
+
+
+def main():
+    if len(sys.argv) != 2:
+        print(USAGE, file=sys.stderr)
+        return
+
+    out_df = pd.DataFrame()
+    header_rows = 5
+    name_template = "Market yield on U.S. Treasury securities at {}   constant"\
```

Comment:
looks like an extra space after {} on the template string

Reply:
That's intentional. The column names have extra spaces.